src/Kanban/index.js

```diff
@@ -280,6 +302,7 @@ class Kanban extends React.PureComponent {
           scrollToColumn={initialColumnIndex}
           verticalStrength={() => {}}
           speed={HORIZONTAL_SCROLL_SPEED}
+          overscanIndicesGetter={overscanIndicesGetter}
```

Comment:
This solution looks good! Do you know if we would need to implement a similar overscan prop for the List component inside SortableList?

Reply:
@Koji23 I do believe we would, but there isn't a default prop for that in list that I found, so it would require a bit of extra work that I'd like to address in a separate PR if possible.